sys/net/gnrc/routing/ipv6_auto_subnets/gnrc_ipv6_auto_subnets.c

```diff
@@ -648,6 +664,18 @@ static void *_eventloop(void *arg)
                 /* send subnet announcement */
                 _send_announce(local_subnets, &timeout_timer, &timeout_msg);
             } else {
+#if CONFIG_GNRC_IPV6_AUTO_SUBNETS_STATIC
```

Comment:
If you used
```suggestion
                if (IS_ACTIVE(CONFIG_GNRC_IPV6_AUTO_SUBNETS_STATIC)) {
```
here instead, would this result in the same binary? Would have the advantage, that this part would be at least compile tested, whenever `gnrc_ipv6_auto_subnets` is included.

Reply:
(you need to remove the parenthesis around `0` in the `#define` of `CONFIG_GNRC_IPV6_AUTO_SUBNETS_STATIC` for this to be compilable in the first place)

Reply:
No need for `IS_ACTIVE` if it's default-defined to 0

Reply:
I can confirm that the same binary is generated.

Reply:
`IS_ACTIVE` has the added benefit that if it is also 0, when the macro is not defined (when for whatever reason in the future >Kconfig :eyes:< the macro is not defined).